client/src/modules/invoices/registry/search.modal.html

```diff
@@ -59,6 +59,32 @@
         </div>
       </div>
 
+      <div class="form-group">
+        <label class="control-label" translate>
+          FORM.LABELS.DEBTOR_GROUP
+        </label>
+
+        <!-- @todo make this a component - it is used everywhere. Calls back method
+             `clear` and resets the id passed to the component -->
+        <span style="display:inline-block;" class="pull-right">
```

Comment:
Same for here!  You can also remove that `@todo` ... we implemented it in `bhClear`.